src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.csproj

```diff
@@ -65,9 +66,6 @@
     <ProjectReference Include="..\..\System.IO\src\System.IO.csproj">
       <TargetGroup>uap101aot</TargetGroup>
     </ProjectReference>
-    <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj">
```

Comment:
Why was this removed? I would expected it to still be necessary. In fact I would kind of expect the System.IO.csproj reference to need to be updated to System.Runtime.Extensions.